src/Tests/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsprojWithCorrectTestRunParameters.cs

```diff
@@ -24,10 +24,9 @@ public GivenDotnetTestBuildsAndRunsTestfromCsprojWithCorrectTestRunParameters(IT
 
         private readonly string[] ConsoleLoggerOutputNormal = new[] { "--logger", "console;verbosity=normal" };
 
-        [Fact(Skip = "flaky")]
```

Comment:
Won't these not run without the [Fact] attribute?

Reply:
You are totally right, my brain must have been broken yesterday. Fixed it. Sorry for wasting your time 😔